Gemfile

```diff
@@ -11,6 +11,11 @@ gem 'rails-i18n', '~> 5'
 gem 'pattern-library', git: 'https://github.com/openstax/pattern-library.git', branch: 'master'
 gem 'bootsnap', require: false
 
+# New Deployments
+gem 'aws-sdk-ssm'
+gem 'dotenv'
```

Comment:
I don't think you needed to add this because it's already there as 'dotenv-rails': https://github.com/openstax/accounts/pull/783/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R156